examples/recipes/joining-multiple-datasources-data/cube.js

```diff
@@ -0,0 +1,27 @@
+const PostgresDriver = require('@cubejs-backend/postgres-driver');
+
+module.exports = {
+  driverFactory: ({ dataSource }) => {
+    if (dataSource === 'suppliers') {
+      return new PostgresDriver({
+        database: 'recipes',
+        host: 'demo-db-recipes.cube.dev',
+        user: 'cube',
+        password: '12345',
+        port: '5432',
+      })
+    }
+
+    if (dataSource === 'products') {
+      return new PostgresDriver({
+        database: 'ecom',
+        host: 'demo-db.cube.dev',
```

Comment:
@rchkv What's the point of having `demo-db-recipes.cube.dev` if we still use `demo-db.cube.dev`? I think we can add another database alongside `recipes` in `demo-db-recipes.cube.dev` and use it here.

Reply:
Make sense! Created a new `ecom` DB in `demo-db-recipes.cube.dev`